components/button/Button.razor.cs

```diff
@@ -174,5 +181,143 @@ private async Task OnMouseUp(MouseEventArgs args)
                 await InvokeAsync(StateHasChanged);
             });
         }
+
+        private string SetButtonColorStyle() => Color switch
+        {
+            ButtonColor.Red1 => Style = "background-color: #fff1f0; border-color: #fff1f0; color: rgba(0,0,0,0.85);",
```

Comment:
Where did you find these colors?

Reply:
They are from here: [Colors](https://ant.design/docs/spec/colors).

Reply:
It's awesome.

Reply:
Please put them into a static Dictionary.

Reply:
Thank you. Done.